doc/doc.md

```diff
@@ -166,8 +167,82 @@ export const settings = {
+| model        | `rgba\|hex`                                               | `rgba`    | The color model to be used                                                                                        |
+| editable     | `boolean`                                                 | true      | If false, disables the field in the UI.                                                                           |
+| options      | `['value']` or `[{label,value}]` or `{label: value, ...}` | undefined | If present, displays a dropdown with the provided options.                                                        |
+| validation   | `function`                                                | undefined | If present, executes function with the new value. Should return a string describing the error or null if no error |
+
+#### Image
+
+| Prop       | Type       | Default   | Description                                                                                                                   |
+| ---------- | ---------- | --------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| type (\*)  | `"image"`  |           | Defines a param of type 'image'                                                                                               |
+| default    | `string`   | undefined | If present, defaults to this value.                                                                                           |
+| multiple   | `string`   | false     | Wether it should accept multiple images                                                                                       |
+| editable   | `boolean`  | true      | If false, disables the field in the UI.                                                                                       |
+| validation | `function` | undefined | If present, executes function with file object to validate. Should return a string describing the error or `null` if no error |
+
+---
+
+### Available Settings
+
+These are the available settings:
+
+| Setting     | Value                       | Default   | Description                                                                                                                                                                                                       |
+| ----------- | --------------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| engine (\*) | `svg\|canvas\|react\|d3`    | undefined | Determines the engine to handle the design function                                                                                                                                                               |
```

Comment:
I feel this gives the impression that it should be a string, and one of those values, but it's a but more complex than that.

Reply:
how would you explain it?